CMakeLists.txt

```diff
@@ -97,18 +116,18 @@ endif (NOT CMAKE_CXX_FLAGS)
 if (NOT CMAKE_BUILD_TYPE)
     # Use debug mode if building in dbg or debug directory.
     get_filename_component(_cmpvar ${CMAKE_BINARY_DIR} NAME)
-    if(_cmpvar STREQUAL "dbg" OR _cmpvar STREQUAL "debug")
+    if("_cmpvar" STREQUAL "dbg" OR "_cmpvar" STREQUAL "debug")
```

Comment:
not sure what went wrong, should have worked
but you can quickly fix it via e.g.

    # unstage last commit
    git reset HEAD^
    # interactive stage stuff back
    git add -p
    # commit force push

you will have to write the commit message again (can be copied from git reflog if you forgot)